### PR TITLE
Separate out Separator from Setting control

### PR DESCRIPTION
**Pull request by jarolrod**
This is required in order to properly support the feature attempted by https://github.com/bitcoin-core/gui-qml/pull/214

### Master

| About | Dev | Prune | Connection | Node |
| ----- | --- | ----- | ---------- | ---- |
| about-master |dev-master |prune-master |connection-master |node-master |

### PR 

| About | Dev | Prune | Connection | Node |
| ----- | --- | ----- | ---------- | ---- |
| about-pr |dev-pr |prune-pr |connection-pr |node-pr |

This also gets us in line with the design file in regards to the height of a setting
| master | pr | design |
| ------ | -- | ------ |
| Screen Shot 2023-02-13 at 7 30 58 PM | Screen Shot 2023-02-13 at 7 21 49 PM | Screen Shot 2023-02-13 at 7 31 54 PM |

[Windows](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/win64/insecure_win_gui.zip?branch=pull/263)
[Intel macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos/insecure_mac_gui.zip?branch=pull/263)
[Apple Silicon macOS](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/macos_arm64/insecure_mac_arm64_gui.zip?branch=pull/263)
[ARM64 Android](https://api.cirrus-ci.com/v1/artifact/github/bitcoin-core/gui-qml/android/insecure_android_apk.zip?branch=pull/263)

### CODE REVIEW

File: src/qml/components/Separator.qml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import "../controls"
+
+Rectangle {
+    height: 1
+    color: Theme.color.neutral5
+}
```
Comment (johnny9) [inline src/qml/components/Separator.qml:12]: github complaining about no newline at eof

Comment (jarolrod) [reply src/qml/components/Separator.qml]: thanks, addressed in 5cbf67f1c479f08cfe3954c56c9962029e6bd5aa

**Comment by jarolrod** [comment]
Updated from from [1ea3922 ](https://github.com/bitcoin-core/gui-qml/commit/1ea392243526ead39d0c660ff54f6886703edd39)to [5cbf67f](https://github.com/bitcoin-core/gui-qml/commit/5cbf67f1c479f08cfe3954c56c9962029e6bd5aa), [compare](https://github.com/bitcoin-core/gui-qml/compare/1ea392243526ead39d0c660ff54f6886703edd39..5cbf67f1c479f08cfe3954c56c9962029e6bd5aa)

changes: addressed no newline at eof

**Review by johnny9 — APPROVED** [review]
ACK 5cbf67f1c479f08cfe3954c56c9962029e6bd5aa

Good change, will rebase the Peers layout on this

**Pull request merged**